### PR TITLE
Add showSaveVideoAs  options

### PR DESCRIPTION
**Pull request by and80506**
add showSaveVideo、 showSaveVideoAs and showCopyVideoAddress options.
image

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -189,6 +207,27 @@ declare namespace contextMenu {
...
 
+		/**
+		Show the `Copy Video Address` menu item when right-clicking on an video.
+
+		@default false
+		*/
+		readonly showCopyVideoAddress?: boolean;
+
+		/**
+		Show the `Save Video` menu item when right-clicking on an video.
```
Comment (sindresorhus) [inline index.d.ts:218]:
```suggestion
		Show the `Save Video` menu item when right-clicking on a video.
```

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
Can you make the fixture video smaller by shortening the duration and maybe making it smaller in dimension? I don't want to waste 1 MB in this repo just on a fixture.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by and80506** [comment]
> Can you make the fixture video smaller by shortening the duration and maybe making it smaller in dimension? I don't want to waste 1 MB in this repo just on a fixture.

OK

**Pull request merged**